src/pages/documentation/release-notes/index.md

```diff
@@ -4,8 +4,188 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## June 8, 2023
+
+### iOS Mobile Core 4.x and compatible extensions
+
+New major version of the Mobile Core SDK for iOS has been released along with updates to other extensions. The current release includes the following changes:
+
+* Updated the minimum supported version to iOS 11.0 and tvOS 11.0 (for extensions which support tvOS).
+* Include XCFrameworks built with Xcode 14.1 with the GitHub release.
+
+Note that all these extensions must be updated together.
+
+* iOS Mobile Core 4.0.0
+* iOS Signal 4.0.0
+* iOS Lifecycle 4.0.0
+* iOS Identity 4.0.0
+* iOS Profile 4.0.0
+* iOS Adobe Experience Platform Assurance 4.0.0
+* iOS Adobe Experience Platform Edge Network 4.0.0
+* iOS Identity for Edge Network 4.0.0
+* iOS Consent for Edge Network 4.0.0
+* iOS Media for Edge Network 4.0.0
+* iOS Edge Bridge 4.0.0
+* iOS Adobe Journey Optimizer 4.0.0
+* iOS Adobe Journey Optimizer - Decisioning 4.0.0
+* iOS Places Service 4.0.0
+* iOS Adobe Analytics 4.0.0
+* iOS Adobe Analytics - Media Analytics for Audio & Video 4.0.0
+* iOS Adobe Target 4.0.0
+* iOS Adobe Campaign Standard 4.0.0
+* iOS Adobe Campaign Classic 4.0.0
+* iOS Adobe Audience Manager 4.0.0
+
+## June 7, 2023
+
+### iOS Target 4.0.0
+
+Major version update for [Adobe Target](../adobe-target/index.md) for Adobe Experience Platform Mobile SDKs on iOS compatible with Mobile Core 4.0.0. The current release includes the following changes:
+
+* Updated the minimum supported version to iOS 11.0.
+* Include XCFrameworks built with Xcode 14.1 with the GitHub release.
+
+### iOS Audience 4.0.0
+
+ Major version update for [Adobe Audience Manager](../adobe-audience-manager/index.md) for Adobe Experience Platform Mobile SDKs on iOS compatible with Mobile Core 4.0.0. The current release includes the following changes:
+
+* Updated the minimum supported version to iOS 11.0.
+* Include XCFrameworks built with Xcode 14.1 with the GitHub release.
+
+### iOS Media 4.0.0
+
+Major version update for [Adobe Analytics - Media Analytics workflows](../adobe-media-analytics/index.md) for Adobe Experience Platform Mobile SDKs on iOS compatible with Mobile Core 4.0.0. The current release includes the following changes:
+
+* Updated the minimum supported version to iOS 11.0 and tvOS 11.0.
+* Include XCFrameworks built with Xcode 14.1 with the GitHub release.
+
+## June 6, 2023
+
+### Android Edge Media 2.0.0 (GA)
```

Comment:
- GA -> new
- General availability -> initial release

Same for the extension release notes